db/README.md

```diff
@@ -0,0 +1,83 @@
+# Academy-of-Cleaning Database
+The AoC Database Instance is deployed on Amazon Relational Database Sysetm (RDS)'s Virtual Private Computer (VPC) in the North America, California Availability Zone (AZ).
+Detailed setup documentation that was followed can be found [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Tutorials.WebServerDB.CreateVPC.html)
+
+`CREATE TABLE` commands can be executed via a Command Line Interface (CLI). The CLI used to initiate these tables was [PgAdmin4](https://www.pgadmin.org/download/)
+
+## Data Overview
+
+#### The following 5 tables are entity tables.
+
+### user
+| Column      | Data Type | Detail |
+| ----------  | --------- | -------|
+| `user_id` | serial | Primary Key, auto-incremented |
+| `oauth_token` | varchar(255) | Generated by OAuth |
+| `email` | varchar(255) | Email of user |
+| `user_type` | varchar(255) | Values = {cleaner, employer} |
+| `is_verified` | boolean | Default: false |
+
+### certification
+| Column      | Data Type | Detail |
+| ----------  | --------- | -------|
+| `cert_id` | serial | Primary Key, auto-incremented |
+| `cert_name` | varchar(255) | Name of the certification |
+| `cert_category` | varchar(255) | e.g. Window Cleaning, Mopping, etc |
+| `is_verified` | boolean | Default: false |
+
+### cleaner_profile
+| Column      | Data Type | Detail |
+| ----------  | --------- | -------|
+| `profile_id` | serial | Primary Key, auto-incremented |
+| `first_name` | varchar(255) | First name of cleaner |
+| `last_name` | varchar(255) | Last name of cleaner |
+| `contact_num` | varchar(50) | Contact number of cleaner e.g. '4161234567' |
```

Comment:
I guess we haven't really discussed the lengths of these different attributes, but in practice it might make a difference to have a contact number of 50 characters vs 10 for validation purposes. maybe something to make a note of.

LGTM overall :)

Reply:
Canada and the US have only 10 character numbers. Other regions, will use more (or maybe less)  
It'll be on the implementation of the backend logic (and the front end) to sanitize and validate the data. So I think 50 characters is fair

Reply:
But also thinking about it... we might want to make it just numerical and then we format the number in a lambda function instead of accepting "(" or "-" 
Its up to you, Christina!

Reply:
Yes, the 50 character logic was due to varying phone number across the world. I was thinking the formatting can be guided via the interface (like have field for area code, then phone number, etc) or be cleaned up in the actual load table functions.